### DISCUSSION TITLE
Book 1 Chapter 13.2 `dev-major`: `Generating Sample Rays`

### DISCUSSION THREAD

**Discussion by LollipopFt** [RayTracing/raytracing.github.io#1075] [Debugging]
https://github.com/RayTracing/raytracing.github.io/blob/dd6660d0f0309c178618695e78c929c1ddcd65f9/books/RayTracingInOneWeekend.html#L3214
was changed to this:
https://github.com/RayTracing/raytracing.github.io/blob/dd6660d0f0309c178618695e78c929c1ddcd65f9/books/RayTracingInOneWeekend.html#L3387-L3390

this flipped the image on the vertical axis such that it is upside down. changing the code to
```diff
            return ray(
                origin + offset,
-                lower_left_corner + s*horizontal + t*vertical - origin - offset
+                lower_left_corner + s*horizontal + (1-t)*vertical - origin - offset
            );
```
fixed it.
is it just me?

**Comment by hollasch** [comment]
Thanks -- created issue #1076
